actionview/test/template/form_helper_test.rb

```diff
@@ -2812,6 +2805,36 @@ def test_button_with_other_formmethod_attribute_and_value
     assert_dom_equal expected, @rendered
   end
 
+  def test_error_with_method_name
+    @post.errors.add(:title, "can't be blank")
+    @post.errors.add(:title, "is too short (minimum is 10 characters)")
+
+    form_for(@post) do |f|
+      concat f.error(:title)
+    end
+
+    expected = whole_form("/posts/123", "edit_post_123", "edit_post", method: "patch") do
+      %(<div>Title can't be blank, Title is too short (minimum is 10 characters)</div>)
```

Comment:
Considering it's close to the field and we usually have a label for the field, do we want to add the field name (`Title`) as part of the error message?